config/psp.cfg

```diff
@@ -0,0 +1 @@
+ppsspp_internal_resolution = "960x544"
```

Comment:
Can we bump the resolution to 1080p or just above that? That is the target resolution I have been going for.
Does "1920x1088" work? (i.e. double) or does it get cut off?

Reply:
It only supports multiples of the original rendering resolution unfortunately. I will test the resolution you mentioned. I'm not sure what it will look like and what the performance will be like.

Do we still care about supporting 720p? I think this is the only supported resolution which will work on that.

The current resolution looks surprisingly good to me on a 1080p screen, though. So if higher resolutions don't work out it should be a pretty good backup.

Reply:
Retroarch will scale the resolution down. It should also maintain aspect ratio.

Reply:
Cool, hope I find time to test it soon then

Reply:
I've tested this now. 1920x1088 looks like on a 1080p tv. I'm feeling some slowdowns with it on my Alienware Steam Machine, but it doesn't show in the FPS meter and it isn't very noticeable. This was when playing Burnout Legends.